nodejs/week1/src/backend/routes/cheap-meals.js

```diff
@@ -0,0 +1,32 @@
+//Respond with the json for all the meals
+//(including it's reviews) that are cheap (you define what a cheap meal is)
+
+const express = require("express");
+const app = express();
```

Comment:
Nice job separating your routes like this. 👍 

One thing to note: since you structured your routes like this, you should use `express.Router()` in this file and your other route files. [(Read about it here) ](https://expressjs.com/en/guide/routing.html#express-router). 

The point is to avoid creating a full express app for every route.  [Here's](https://stackoverflow.com/a/33261362) a good explanation:
> When `var app = express()` is called, an `app` object is returned. Think of this as the **main app**.
> When `var router = express.Router()` is called, a slightly different **mini app** is returned. The idea behind the mini app is that each route in your app can become quite complicated, and you'd benefit from moving all that code into a separate file. Each file's router becomes a mini app, which has a very similar structure to the main app.

This would require 3 steps for each route: 

1. Create a route handler in your rout file like below (and rename `app` in it to `router` to denote it's different from the main app `app` in server.js): 
```
// const app = express(); // Replace this with the line below
const router = express.Router();
```

2. In the route file, rename the route to "/" like below: 
```
/*
// Replace this: 
app.get("/cheapMeal", function (request, response) {
  response.send(cheapMeal);
});
*/

// with this:
router.get("/", (request, response) => response.send(cheapMeal));

``` 

3. In server.js, replace `app.get()` with `app.use()` like this: 

```
/*
// Replace this
const cheapMeal = require("./routes/cheap-meals");
app.get("/cheapMeal", cheapMeal);
*/

// with this:
const cheapMealRouter = require("./routes/cheap-meals");
app.use("/cheap-meals", cheapMealRouter); // notice APP.USE here!
```

Reply:
implemented this